projects/material-addons/src/lib/quick-list/quick-list-compact/quick-list-compact.component.ts

```diff
@@ -4,7 +4,8 @@ import { BaseQuickListComponent, QuickListItem } from '../base-quick-list.compon
 @Component({
   selector: 'mad-quick-list-compact',
   templateUrl: './quick-list-compact.component.html',
-  styleUrls: [],
+  styles: [`.quick-list-row{flex-direction: row; box-sizing: border-box; display: flex}`],
```

Comment:
Ließe sich das nicht auch mithilfe dieser Klassen im flex-utility.scss lösen?

Reply:
flex-utility ist nur in demo app
projects/material-addons/src/lib kann das nicht verwenden